study-builder/studies/circadia/study.conf

```diff
@@ -0,0 +1,672 @@
+{
+  "tenant": {
+    "domain": ${auth0.domain},
+    "mgmtClientId": ${auth0.mgmtClientId},
+    "mgmtSecret": ${auth0.mgmtSecret}
+  },
+
+  "umbrella": {
+    "name": "circadia",
+    "guid": "circadia"
+  },
+
+  "study": {
+    "name": "circadia",
+    "guid": "circadia",
+    "studyEmail": "cancergenomecollective@broadinstitute.org",
+    "baseWebUrl": ${baseWebUrl},
+    "irbPassword": ${irbPassword},
+    "plusCodePrecision": "MEDIUM",
+    "shareParticipantLocation": true
+  },
+
+  "supportedLanguages": [
+    {
+      "language": "en",
+      "name": "English",
+      "isDefault": true
+    }
+  ],
+
+  "statistics": [],
+
+  "client": {
+    "name": "circadia-angular-client",
```

Comment:
`name` is actually deprecated. See `study-builder/studies/study-example.conf` file.